probe-rs/src/debug/debug_info.rs

```diff
@@ -2084,4 +2129,60 @@ mod test {
 
         insta::assert_snapshot!(printed_backtrace);
     }
+    #[test]
+    fn probe_rs_debug_unwind_tests() {
+        // TODO: Add RISC-V tests.
+        for chip_name in ["nRF52833_xxAA", "RP2040"] {
+            let debug_info = debug_info(format!("debug-unwind-tests/{chip_name}.elf").as_str());
+            let coredump = fs::read(get_path_for_test_files(
```

Comment:
You can use `load_path` to save yourself the `fs::read` :)

Reply:
Done